CodeEdit/Features/Acknowledgements/Views/AcknowledgementsView.swift

```diff
@@ -14,38 +14,55 @@ struct AcknowledgementsView: View {
     @ObservedObject
     private var model = AcknowledgementsViewModel()
 
+    @State private var displayDivider = false
+
     var body: some View {
-        VStack {
+        VStack(spacing: 0) {
             Image(nsImage: NSApp.applicationIconImage)
                 .resizable()
-                .frame(width: 70, height: 70)
+                .frame(width: 48, height: 48)
             Text("Acknowledgements")
-                .font(.largeTitle)
+                .font(.title)
                 .fontWeight(.bold)
-            ScrollView {
-                ForEach(model.acknowledgements, id: \.name) { acknowledgement in
-                    HStack {
-                        Text(acknowledgement.name)
-                            .font(.title3)
-                            .fontWeight(.semibold)
-
-                        Spacer()
-
-                        Button {
-                            openURL(acknowledgement.repositoryURL)
-                        } label: {
-                            Image(systemName: "arrow.right.circle.fill")
-                                .foregroundColor(Color(nsColor: .tertiaryLabelColor))
+                .padding(.vertical, 8)
+            Divider()
+                .opacity(displayDivider ? 1 : 0)
+            OffsettableScrollView(showsIndicator: false) { offset in
+                displayDivider = offset.y < 0
+            } content: {
+                LazyVStack(spacing: 0) {
+                    ForEach(
+                        Array(zip(model.acknowledgements.indices, model.acknowledgements)),
+                        id: \.1.name
+                    ) { (index, acknowledgement) in
+                        if index != 0 {
+                            Divider()
+                                .frame(height: 0.5)
+                                .opacity(0.5)
+                        }
+                        HStack {
```

Comment:
Maybe move this to a `private func acknowledgementCell(_ acknowledgement: )` to make it easier to read

Reply:
I did the same thing that you did with the contributors window.